packages/components/src/internal/components/domainproperties/assay/AssayDesignerPanels.spec.tsx

```diff
@@ -317,6 +316,8 @@ describe('AssayDesignerPanels', () => {
         );
     });
 
+    // FIXME: This test case triggers issues with React Beautiful DND, I think there is a way to put that library in
+    //  test mode. We maybe accidentally disabled that in the test stabilization PR.
```

Comment:
I believe it was this line that was in the `initUnitTestMocks` function:
`window['__react-beautiful-dnd-disable-dev-warnings'] = true;`

Reply:
Thanks! I'll see about adding that to our jest config before I merge this